src/stores/Frame/Region/RegionSetStore.ts

```diff
@@ -298,6 +298,9 @@ export class RegionSetStore {
             if (region === this.selectedRegion) {
                 this.selectedRegion = this.regions[0];
             }
+            const selectedInd = this.regions.findIndex(r => r === region);
+            let exportRegionIndexes = FileBrowserStore.Instance.exportRegionIndexes.filter(x => x !== selectedInd).map(x => (x > selectedInd ? x - 1 : x));
```

Comment:
do we need to update the deleteExportRegionIndex function in the FileBrowserStore, so that some indexes are reduced by 1?

Reply:
No. If I understand correctly. The deleteExportRegionIndex function is to unselect export regions, and the regions still exist. In the current structure, there is no need to reduce some indexes by 1 when unselecting the exporting regions.

Reply:
Looks like we don't need to adjust deleteExportRegionIndex.